data/projects/coala_cli.md

```diff
@@ -19,32 +19,84 @@ project is going to rely heavily upon mockups and feedback from the
 community. Also, in addition to the above changes the applicant is encouraged
 to come up with their own ideas to improve the CLI.
 
-[cEP-0004](https://github.com/coala/cEPs/pull/29/files) contains further
-details about the project. It is incomplete though and open to a lot of
-improvement.
+[cEP-0010](https://github.com/coala/cEPs/pull/79/files) contains further
+details about the project.
+Make coala easier to use, hence the project is going to rely heavily upon
+mockups and feedback from the community. Adds new functionality so user could
+have multiple options when uses coala. The user interface would help coala
+become much more user friendly and intuitive.
+
 
 #### MILESTONES
 
+[BONDING] https://gitlab.com/coala/GSoC-2017/milestones/16
+[PHASE 1] https://gitlab.com/coala/GSoC-2017/milestones/17
+[PHASE 2] https://gitlab.com/coala/GSoC-2017/milestones/18
+[PHASE 3] https://gitlab.com/coala/GSoC-2017/milestones/19
+
 ##### PREPARATION/BONDING
 
-* The [cEP](https://github.com/coala/cEPs/pull/29/files) has been updated, finished and merged with
+* The [cEP](https://github.com/coala/cEPs/pull/79/files) has been updated, finished and merged with
   the help of mockups and community feedback.
 * A survey is done on the importance of each proposed change to the CLI.
 * Concept for implementation exists in the cEP.
 
+Related issues:
+
+[Google Form] https://gitlab.com/coala/GSoC-2017/issues/54
+[cEP] https://gitlab.com/coala/GSoC-2017/issues/52
+[mockups] https://gitlab.com/coala/GSoC-2017/issues/53
+
+
 ##### CODING PHASE 1
```

Comment:
you could make these headers link to the corresponding milestones on gitlab, it wouldnt be a bad idea